### ISSUE TITLE
routes are not integrated into the main routes

### ISSUE THREAD

**Issue by Chumper** [wardrobecms/core-archived#12]
I installed the cms with the theme blocky.
If i try to visit the about page i get a 404, so the routes.php is not integrated into the main routes file.

**Comment by rtablada** [comment]
Thank you for the reminder I've been meaning to get to this feature and forgot about it. Will do.

**Comment by Chumper** [comment]
How about a simple routing feature?
Let's say all pages in the folder 'pages' will be routed by default.
In this case you don't have to implement own routes.

I would suggest a page controller which will look for md files in the folder and if found display them.

**Comment by ericlbarnes** [comment]
Not sure I like that because it would limit theming each page. 

Maybe a work around could be to have any blade file be routed unless it starts with an underscore. For instance your theme structure could be: 

/post.blade.php
/about.blade.php
/_header.blade.php

Then if the route throws a 404 it will make one last attempt to see if a file exists in the theme before bailing.

**Comment by Chumper** [comment]
In my opinion it would not, maybe you missunderstood me.
Just add to the current theme folder (e.g. `blocky`) a new one called `pages`.
For the `blocky` theme i would put the `about.blade.php` in there and i wouldn't have to add a custom route with the routes.php
So it will only look in the `pages` folder when a 404 arises if there is a page to route to.

**Comment by ericlbarnes** [comment]
Yeah we are pretty close then. Where I was talking about being limited was this: 

> I would suggest a page controller which will look for md files in the folder and if found display them.

I think the advantage of not having them in pages/ is that it would allow full paths. Example: 

/about.blade.php
/post.blade.php
/company/profile.blade.php
/etc...

So then it would auto route: .com/about, .com/company/profile, etc...

The advantage in mind is that it's a one to one mapping instead of prefixing the pages/ directory. That way new users to the system instantly see it. Probably a moot point. 

Of course I am just brain storming. I haven't looked into the feasibility of all this. 

Also just so you know more about why I originally added routes to themes was because I figured some sites might not like the default structure. /post/:slug, /archive, etc. So they could use other paths to suit their sites or to suit their locale. Some of the pages was added just to demonstrate what could be done.
